docs/how_to_add_a_publisher.md

```diff
@@ -242,29 +245,28 @@ Now bring your parser to life and define the attributes you want to extract.
 
 One important caveat to consider is the type of content on a particular page.
 Some news outlets feature live tickers, displaying podcasts, or hub sites that link to other pages but are not articles themselves.
-For now, you need not worry about handling non-article sites. 
-Your parser should focus on extracting the desired attributes from most pages you would consider as articles.
-Sites that do not contain the desired attributes will be filtered by the library in a later stage of the pipeline.
+At this stage, there's no need to concern yourself with handling non-article websites. 
```

Comment:
In my opinion, the word "websites" could be replaced by "page". In my mind, websites refers to multiple top level domain.

Reply:
Good point, i will replace existing occasions of `websites`.